Login Demo/Info.plist

```diff
@@ -36,8 +18,47 @@
 	<string>1.0</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLName</key>
+			<string>com.firebase.$(PRODUCT_NAME:rfc1034identifier)</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>com.firebase.$(PRODUCT_NAME:rfc1034identifier)</string>
+			</array>
+		</dict>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLName</key>
+			<string>FacebookURL</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>fb&lt;YOUR FACEBOOK APP ID&gt;</string>
+			</array>
+		</dict>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLName</key>
+			<string>GoogleURL</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>&lt;YOUR GOOGLE REVERSE CLIENT ID&gt;</string>
+			</array>
+		</dict>
+	</array>
 	<key>CFBundleVersion</key>
 	<string>1</string>
+	<key>Dummy</key>
+	<string>NOTE: Expand the URL types below and configure &quot;&lt;YOUR FACEBOOK APP ID&gt;&quot;</string>
```

Comment:
It's more clear that they are different schemes, since technically they are different schemes, so I broke them out into multiple schemes, rather than calling the bundle scheme the "master set of schemes"

I also named the URL schemes and changed the message a little more to hopefully be more clear.